lib/tasks/mastodon.rake

```diff
@@ -512,14 +512,9 @@ namespace :mastodon do
 
     desc 'Remove cached remote media attachments that are older than NUM_DAYS. By default 7 (week)'
     task remove_remote: :environment do
-      time_ago = ENV.fetch('NUM_DAYS') { 7 }.to_i.days.ago
-      nb_media_attachments = 0
-
-      MediaAttachment.where.not(remote_url: '').where.not(file_file_name: nil).where('created_at < ?', time_ago).select(:id).reorder(nil).find_in_batches do |media_attachments|
-        nb_media_attachments += media_attachments.length
-        Maintenance::UncacheMediaWorker.push_bulk(media_attachments.map(&:id))
-      end
-      puts "Scheduled the deletion of #{nb_media_attachments} media attachments"
+      require_relative '../mastodon/media_cli'
+      cli = Mastodon::MediaCLI.new([], days: ENV['NUM_DAYS'] || 7)
```

Comment:
I think that ".to_i" is requires for "['NUM_DAYS']".

However, I still feel that it is not working properly.

I executed next line.
<code>RAILS_ENV=production NUM_DAYS=60 bundle exec rake mastodon:media:remove_remote</code>

After that, when I executed next line,  multiple jobs were added to Sidekiq.
<code>RAILS_ENV=production bundle exec bin/tootctl media remove --days 60 --background</code>